core/src/scene/sceneLoader.cpp

```diff
@@ -78,31 +78,43 @@ bool SceneLoader::loadConfig(const std::string& _sceneString, Node& root) {
     return true;
 }
 
-void SceneLoader::applyUpdates(Node& root, const std::vector<SceneUpdate>& updates) {
+void SceneLoader::applyUpdate(Node& root, const std::vector<std::string>& keys, Node value) {
 
-    for (const auto& update : updates) {
-
-        auto keys = splitString(update.keys, COMPONENT_PATH_DELIMITER);
+    auto node = root;
+    std::string key;
+    for (auto it = keys.begin(); it != keys.end(); ++it) {
+        key = *it;
+        if (it + 1 == keys.end()) { break; } // Stop before last key.
+        node.reset(node[key]); // Node safely becomes invalid is key is not present.
+    }
 
-        auto node = root;
-        std::string key;
-        for (auto it = keys.begin(); it != keys.end(); ++it) {
-            key = *it;
-            if (it + 1 == keys.end()) { break; } // Stop before last key.
-            node.reset(node[key]); // Node safely becomes invalid is key is not present.
+    if (node) {
+        try {
+            node[key] = value;
+        } catch (YAML::ParserException e) {
+            LOGW("Cannot update scene - value invalid: %s", e.what());
         }
+    } else {
+        std::string path;
+        for (const auto& k : keys) { path += "." + k; }
+        LOGW("Cannot update scene - key not found: %s", path.c_str());
+    }
+}
 
-        if (node) {
-            try {
-                node[key] = YAML::Load(update.value);
-            } catch (YAML::ParserException e) {
-                LOGW("Cannot update scene - value invalid: %s", e.what());
-            }
-        } else {
-            std::string path;
-            for (const auto& k : keys) { path += "." + k; }
-            LOGW("Cannot update scene - key not found: %s", path.c_str());
-        }
+void SceneLoader::applyUpdates(Scene& scene, const std::vector<SceneUpdate>& updates) {
+    auto& root = scene.config();
+    for (const auto& update : updates) {
+        scene.removeGlobalUpdate(update.keys);
+        auto keys = splitString(update.keys, COMPONENT_PATH_DELIMITER);
+        auto valueNode = YAML::Load(update.value);
```

Comment:
`YAML::Load` throws exceptions on parsing errors, I think we'll need a try/catch here :\